spritz/spritz-android/src/main/java/com/novoda/spritz/SpritzPageWithOffset.java

```diff
@@ -0,0 +1,72 @@
+package com.novoda.spritz;
+
+import android.animation.TimeInterpolator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class SpritzPageWithOffset {
+
+    private final long autoPlayDuration;
+    private final long autoPlayEnd;
+    private final long swipeEnd;
+    private final TimeInterpolator swipeForwardInterpolator;
+    private final TimeInterpolator swipeBackwardsInterpolator;
+
+    private SpritzPageWithOffset(long autoPlayDuration,
+                                 long autoPlayEnd,
+                                 long swipeEnd,
+                                 TimeInterpolator swipeForwardInterpolator,
+                                 TimeInterpolator swipeBackwardsInterpolator) {
+
+        this.autoPlayDuration = autoPlayDuration;
+        this.autoPlayEnd = autoPlayEnd;
+        this.swipeEnd = swipeEnd;
+        this.swipeForwardInterpolator = swipeForwardInterpolator;
+        this.swipeBackwardsInterpolator = swipeBackwardsInterpolator;
+    }
+
+    long autoPlayDuration() {
+        return autoPlayDuration;
+    }
+
+    long autoPlayEnd() {
+        return autoPlayEnd;
+    }
+
+    long swipeEnd() {
+        return swipeEnd;
+    }
+
+    TimeInterpolator swipeForwardInterpolator() {
+        return swipeForwardInterpolator;
+    }
+
+    TimeInterpolator swipeBackwardsInterpolator() {
+        return swipeBackwardsInterpolator;
+    }
+
+    static List<SpritzPageWithOffset> fromSpritzPages(SpritzPage... spritzPages) {
```

Comment:
Not sure this should be here, I am not a fan of a `(List<ModelA>) -> List<ModelB>` sitting in `ModelB`. Would move to another class.